onnxruntime/core/providers/cpu/nn/layer_norm_impl.h

```diff
@@ -65,6 +65,7 @@ class LayerNormImpl : public OpKernel {
   const bool contrib_op_;
   mutable IAllocatorUniquePtr<float> scale_fp32_;
   mutable IAllocatorUniquePtr<float> bias_fp32_;
+  mutable OrtMutex mutex_;
```

Comment:
There is no need to use mutex, since after prepack, the variables scale_fp32_ and bias_fp32_  are read only during inference.